src/lib.rs

```diff
@@ -12,3 +12,16 @@ mod tlsf;
 pub use llff::Heap as LlffHeap;
 #[cfg(feature = "tlsf")]
 pub use tlsf::Heap as TlsfHeap;
+
+/// Init a heap with a static memory region.
+#[macro_export]
```

Comment:
Since this is a public API, it needs a bit more documentation, preferably with an example. I think it also needs a CHANGELOG.md entry.